wallet/wallet.go

```diff
@@ -65,6 +65,16 @@ var (
 	// down.
 	ErrWalletShuttingDown = errors.New("wallet shutting down")
 
+	// errUnknownTransaction is returned when an attempt is made to label
+	// a transaction that is not known to the wallet.
+	errUnknownTransaction = errors.New("cannot label transaction not " +
```

Comment:
Similar comment here re returning non-exported errors, which means that users need to use string matching rather than comparing against the variable directly.